src/components/MissionVision.jsx

```
@@ -0,0 +1,218 @@
+    transition: {
+      duration: 0.5,
+      ease: "easeOut",
+    },
+  },
+};
+
+const MissionVision = () => {
+  const [openIndex, setOpenIndex] = useState(null);
+  const [activeAcc, setactiveAcc] = useState(null);
+  const [isPlaying, setIsPlaying] = useState(false);
+  const [isHovered, setIsHovered] = useState(true);
```

ESLint

[error] 'isHovered' is assigned a value but never used.

(no-unused-vars)